src/solutions/forms.py

```diff
@@ -47,7 +47,7 @@ def clean_file(self):
                     zip = zipfile.ZipFile(data)
                     if zip.testzip():
                         raise forms.ValidationError(_('The zip file seems to be corrupt.'))
-                    if sum(fileinfo.file_size for fileinfo in zip.infolist()) > max_file_size:
+                    if sum(fileinfo.file_size for fileinfo in zip.infolist()) > (max_file_size * min(len(zip.infolist()), 8)):
```

Comment:
You changed `max(len(zip.infolist()), 8)` to `min(len(zip.infolist()), 8)`. 
That's okay now :)